test/CodeReview.Orchestrator.SubsystemTests/StartUpFixture.cs

```diff
@@ -23,14 +22,18 @@ public void Dispose()
             if (Directory.Exists(Config.OutputDirectoryPath))
                 Directory.Delete(Config.OutputDirectoryPath, true);
         }
-        
+
         private static IConfiguration GetConfiguration()
         {
             var builder = new ConfigurationBuilder()
                 .SetBasePath(Directory.GetCurrentDirectory())
                 .AddEnvironmentVariables()
                 .AddJsonFile("appsettings.json", false, false);
-            
+
+            #if AZURE
```

Comment:
@pNovikau usually this kind of problems using environment variables which contain environment name. Suggested approach eliminates usage of pre-processor directives.